### PR TITLE
Allow includes and extend tags to be used in a template

### PR DESCRIPTION
**Pull request by TayHobbs**

### CODE REVIEW

**Comment by TayHobbs** [comment]
@aromanovich this is failing because of the issues addressed in #4

File: tests/func_tests/test_inheritance.py [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+        'var': Dictionary(
+            {'x': Scalar(label='x', linenos=[1]), 'y': Scalar(label='y', linenos=[1])},
+            label='var',
+            linenos=[1]
+        ),
+        'more':  Scalar(label='more', linenos=[2]),
+    })
+    assert struct == expected_struct
+
+def test_include_override_1():
```
Comment (aromanovich) [inline tests/func_tests/test_inheritance.py]: It think at least this test case is wrong. If we render ``inner_include_override_1.html`` with an empty context, both parent's and child's blocks are rendered and the output is the following:
```
<p>I am the default, my name is </p>
<p>Hello, </p>
```
Which means that both ``name`` and ``default_name`` must be present in the expected structure.

As far as I know, Jinja2 does not allow to override blocks of included templates.

**Comment by TayHobbs** [comment]
@aromanovich I could only think of two test cases for `include` after reading your note. If you think of more please let me know.

**Comment by aromanovich** [comment]
No, everything else seems to be correct.

I've released your changes on PyPI: https://pypi.python.org/pypi/jinja2schema/0.1.2. Thank you for the contribution! :)

**Pull request merged**